src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,19 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
+import { UserInfo } from '../UserInfo';
+import { Todo } from '../../types/todo';
+
+type Props = {
+  todo: Todo
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => (
+  <article
+    className={classNames('TodoInfo', {
+      'TodoInfo--completed': todo.completed === true,
```

Comment:
@Koliras comment makes sense

```suggestion
      'TodoInfo--completed': todo.completed,